include/bluetooth/hci_vs_sdc.h

```diff
@@ -392,6 +392,28 @@ int hci_vs_sdc_scan_accept_ext_adv_packets_set(
  */
 int hci_vs_sdc_set_role_priority(const sdc_hci_cmd_vs_set_role_priority_t *params);
 
+/** @brief Set Event Start Task.
+ *
+ * For the complete API description, see sdc_hci_cmd_vs_set_event_start_task().
+ *
+ * @param[in]  params Input parameters.
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int hci_vs_sdc_set_event_start_task(const sdc_hci_cmd_vs_set_event_start_task_t *params);
+
+/** @brief Connection Anchor_Point Update Event Reports enable.
```

Comment:
Anchor Point or Anchor-Point?

Reply:
Here I used what we use here: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/softdevice_controller/include/sdc_hci_vs.h#L1726